drivers/nrf9160_gps/nrf9160_gps.c

```diff
@@ -158,6 +179,27 @@ static void gps_thread(int dev_ptr)
 
 		switch (raw_gps_data.data_id) {
 		case NRF_GNSS_PVT_DATA_ID:
+			if (gps_is_blocked(&raw_gps_data.pvt)) {
```

Comment:
I think it would make more sense to just concatinate this error and the pvt_deadline_missed error. The result of both are an invalid PVT frame.

Reply:
I'd like to keep them separated to be able to give feedback to the user that the GPS is blocked due to the LTE link parameters. Then they can adjust their link config or go offline to make it work better. If the deadline is missed, there's not much to do, though.

Reply:
Isn't this a runtime problem? Can you make a link config that works any environment?

Reply:
Unfortunately, it seems to not be possible to configure eDRX/PSM in a way that will work reliably on all networks. The only safe way to know GPS will get time to run, at least with the driver as it is today, is to go offline. In the future we'll implement the option to prioritize GPS over LTE, but too late for the release.
So the best I was able to figure out for now is to do this check, not pass on known bad PVT frames, and print an explainatory debug message.